frontend/packages/react-form-wizard/src/Section.tsx

```diff
@@ -98,10 +98,14 @@ function SectionInternal(props: SectionProps) {
                             )}
                           </div>
                         </Split>
-                        {expanded && props.description && (
-                          <Text component="small" style={{ paddingTop: 8 }}>
+                        {/* TODO this causes react minified error in prod.. */}
```

Comment:
Did you have to patch this onto a cluster to see it? Or how were you running?

Reply:
I was running using `npm run plugins`. And would see a react minified error from this line. Not sure why as the <Content> component works just fine elsewhere.